pkg/security/scc/byrestrictions.go

```diff
@@ -54,7 +83,7 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 // than Secret, ConfigMap, EmptyDir, DownwardAPI, Projected, and None will result in
 // a score of 5.  If the SCC only allows these trivial types, it will have a
 // score of 0.
-func volumePointValue(scc *securityapi.SecurityContextConstraints) int {
+func volumePointValue(scc *securityapi.SecurityContextConstraints) weight {
```

Comment:
Now the code starting to looks a bit strange because we're using points and weight in different places with the similar meaning. Let me know if I need to fix it also.